OSDOCS-15874: adds physically bound image building to bootc docs Micr… (#97670)
openshift/openshift-docs: modules/microshift-embed-cont-images-bootc-image.adoc

// Module included in the following assemblies:
//
// microshift_install_bootc/microshift-install-bootc-physically-bound.adoc

:_mod-docs-content-type: PROCEDURE
[id="microshift-embed-cont-images-bootc-image_{context}"]
= Embedding container images into a bootc image

> First, you must add instructions to an existing Containerfile to copy the images you want and list them in an accessible place where you can keep track of the image names. Then, you must copy images locally from the `/usr/lib/containers/storage/` directory to the local container storage.

Reviewer: ```suggestion
First, you must add instructions to an existing Containerfile to copy the images you want and list them in a file to keep track of the copied image names. Then, you must copy images locally from the `/usr/lib/containers/storage` directory to the local container storage.
```
